ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/QuestDesignerLariat.asset

```diff
@@ -0,0 +1,25 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d8cfffae36be18b40b59e8e84d47a6ad, type: 3}
+  m_Name: QuestDesignerLariat
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: 10299a39b72aab747a9b16ceee7f8742, type: 2}
+  positionName: Quest Narrative Designer
+  jobSummaryLines:
+  - We are looking for a passionate narrative designer!
+  - 'Requirements for this position:'
+  jobReqLines:
+  - '>Passion in video games'' narrative and storytelling'
```

Comment:
I would rephrase this as "Passion for narrative and storytelling in videogames".

Reply:
^